### ISSUE TITLE
pre-trained weights

### ISSUE THREAD

**Issue by duanyuejing** [fsmis01/CoW#6]
May I ask which of the pre-trained weights is the best indicator?

**Comment by yuqiushi55** [comment]
Due to the absence of a validation set, the default is 50000.pth. Additionally, may I ask if setting 2 can be reproduced?

**Comment by duanyuejing** [comment]
> Due to the absence of a validation set, the default is 50000.pth. Additionally, may I ask if setting 2 can be reproduced?

The readme file states that there are two pre-trained weights: "Download the vanilla version or the deeplabv3 version of the pre-trained ResNet-101 weights and place your checkpoints folder". I'm not sure which one of the optimal parameters 84.10 used in the paper is.

**Comment by duanyuejing** [comment]
> Due to the absence of a validation set, the default is 50000.pth. Additionally, may I ask if setting 2 can be reproduced?

Currently, I am replicating set1, but set2 has not been tested yet.

**Comment by yuqiushi55** [comment]
deeplabv3_resnet101_coco-586e9e4e.pth。I can reproduce set 1, but not set 2. If you can reproduce set 2, please teach me。

**Comment by duanyuejing** [comment]
> deeplabv3_resnet101_coco-586e9e4e.pth。I can reproduce set 1, but not set 2. If you can reproduce set 2, please teach me。

Can you reach the "Abd-MRI" Mean value of 84.10 as specified in Set 1? If I can reproduce Set2, I will let you know in time.

**Comment by yuqiushi55** [comment]
The dataset is "Abd-MRI". In set 1, the five-fold cross-validation results for the four organs are 0.802127, 0.902741, 0.841259431, and 0.765456104. The reproduced results mentioned above are slightly lower than those in the paper.

**Comment by jcjiang99** [comment]
> May I ask which of the pre-trained weights is the best indicator?
Resnet101 pretrained on MS-COCO

**Comment by jcjiang99** [comment]
> Due to the absence of a validation set, the default is 50000.pth. Additionally, may I ask if setting 2 can be reproduced?

Of course. We have corresponding weights.

**Comment by yuqiushi55** [comment]
Dear author, is the setting of the right and left kidneys in the case of set 2 correct: TEST_LABEL=[2,3]、
EXCLUDE_LABEL=[2,3]? Liver and spleen ：TEST_LABEL=[1,4] 、EXCLUDE_LABEL=[1,4]？

**Comment by jcjiang99** [comment]
> Dear author, is the setting of the right and left kidneys in the case of set 2 correct: TEST_LABEL=[2,3]、 EXCLUDE_LABEL=[2,3]? Liver and spleen ：TEST_LABEL=[1,4] 、EXCLUDE_LABEL=[1,4]？

Yes, but this kind of setting is traditional manner and the new manner is TEST_LABEL=[1,2,3,4], EXCLUDE_LABEL=[1,2,3,4], namely excluding and testing all classes at the same time.

**Comment by yuqiushi55** [comment]
Thank you, author. Could you please provide the Abd-CT and CMR datasets? My email is 17851974812@163.com

**Comment by jcjiang99** [comment]
> Thank you, author. Could you please provide the Abd-CT and CMR datasets? My email is [17851974812@163.com](mailto:17851974812@163.com)

Sure, I will send you in recent days.

**Comment by yuqiushi55** [comment]
Thank you, author. You are the best person in the world. I wish you success in your scientific research.
